dispatch/static/manager/src/js/index.js

```diff
@@ -53,6 +53,8 @@ render((
           <Route path='/login' component={Pages.Login} />
         </Route>
 
+        <Route path='/logout' component={Pages.Logout} />
```

Comment:
I think i put this in the right place, if it should be another level deeper or w.e lmk but i think this looks right.